### DISCUSSION TITLE
Hello! The plugin is very cool! Thank you!

### DISCUSSION THREAD

**Discussion by ThunderWeed** [nothingislost/obsidian-hover-editor#195] [General]
Hello! The plugin is very cool! Thank you!

**Comment by pjeby** [comment]
Please put bug reports in the Issues section, not discussions.  That way they can be tracked and resolved with the other issues. Thanks!

**Reply by ThunderWeed** [reply]
Sorry, I'll fix it now. Haven't used this site before..)
